angular-konami.js

```diff
@@ -17,7 +20,7 @@ angular.module('konami', [])
         }
 
         // Let user define a custom code.
-        var konamiKeys = attr.konamiKeys || konamiKeysDefault;
+        var konamiKeys = konamiKeys || konamiKeysDefault;
```

Comment:
Hey!  no worries. Basically instead of using attr it is using an isolated scope which also allows for javascript objects to be passed, as well as strings. I ran into this when I was using another directive that called this one. In its current state if I had the following it would work:

    <div controller="controller as ctrl">
        <div konami-code="ctrl.foo()" konami-keys="[38,38]"></div>
    </div>

However if I did the following it would not work since the konami-angular.js sees it as a string rather than an array:

    (in Controller)
    ctrl.keys = [38,38];     
    ...
    (In View)
    <div controller="controller as ctrl">
        <div konami-code="ctrl.foo()" konami-keys="ctrl.keys"></div>
    </div>

Making it an isolated/bound scope it should allow for dynamic swapping of the expected keys, and allows objects to be passed in. If they don't provide a konamiKey then it should just default to your konamiKeysDefault if I am not mistaken~ 

Hopefully that all made sense if not I can show you the code example I am working with.